dockerfiles/theia/README.md

```diff
@@ -3,9 +3,16 @@
 ## Build image manually
 Example:
 using build script:
+
+```
 ./build.sh --build-args:GITHUB_TOKEN=$GITHUB_TOKEN,THEIA_VERSION=0.3.10 --tag:0.3.10-nightly
+```
+
 with native docker:
+
+```
```

Comment:
I think given version in the example won't work because plugin model is released in `0.3.12`. Please add note and correct example.